src/mishap.js

```diff
@@ -0,0 +1,12 @@
+const { execQuery } = require('./db');
+
+const MISHAP_CREATE = 'INSERT INTO dispatchMishaps (date, mishap) VALUES (?, ?)';
```

Comment:
Change table name after you change it in the schema and the DB itself :)

Reply:
Completed